tools/webots.py

```diff
@@ -40,6 +41,10 @@ def register(command):
         default=True,
         help="Disables cleaning the build volume before compiling",
     )
+    #
```

Comment:
I think you might have forgotten to finish this comment - or maybe it was a WIP until the branch is finished 🤷🏿‍♀️